dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem.cpp

```diff
@@ -7,15 +7,32 @@
 #include "RuntimeProfiler.h"
 #include "ResourceAccessor.h"
 
+winrt::IMap<winrt::hstring, winrt::RadioMenuFlyoutItem> RadioMenuFlyoutItem::s_selectionMap = nullptr;
```

Comment:
>IMap [](start = 7, length = 4)

Don't use a WinRT object in a static. this can just be a std::map not a winrt::IMap.

Reply:
fixed